js/jquery-smartphoto.js

```diff
@@ -3477,6 +3462,45 @@ var SmartPhoto = function (_ATemplate) {
       _this._doAnim();
     }, _this.data.forceInterval);
 
+    // IF history API is enabled
+    // > setup history event listener
+    if (_this.data.useHistoryApi && _this.data.historyApiMethod === 'push') {
+
+      window.onpopstate = function (event) {
```

Comment:
Sorry for the late review
Please not to use onpopstate function as it self, I prefer using window.addEventlistener("popstate") instead